sympy/stats/tests/test_continuous_rv.py

```diff
@@ -351,6 +356,7 @@ def test_exponential():
     assert P(X > 0) == S(1)
     assert P(X > 1) == exp(-rate)
     assert P(X > 10) == exp(-10*rate)
+    assert quantile(X, p) == Intersection(Interval(0, oo), FiniteSet(-log(1 - p)/rate))
```

Comment:
`quantile(X)(x)` is a not a function of `x`. Maybe it will make more sense to define `quartiles` on the basis of these. I am not sure though
Or maybe we can make `quantile(X)(p)`. It is feasible, I think

Reply:
I looked into it and actually `quantile(X)(x)` or `quantile(X)(p)` won't make much sense.

Reply:
The quantile function accepts one parameter.